app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from "react";
+
+import { Formik, Form } from "formik";
+import { Button, Pane, Toastr } from "neetoui/v2";
+import { Input, Select } from "neetoui/v2/formik";
+
+import formValidationSchemas from "constants/formValidationSchemas";
+
+import { SELECT_OPTIONS } from "./constants";
+
+export default function ContactForm({ isEdit, contact, onClose }) {
+  const [submitted, setSubmitted] = useState(false);
+  const handleSubmit = () => {
+    onClose();
+    Toastr.success("Contact was successfully created.");
+  };
+
+  return (
+    <Formik
+      initialValues={contact}
+      onSubmit={handleSubmit}
+      validateOnBlur={submitted}
+      validateOnChange={submitted}
+      validationSchema={formValidationSchemas.contactsForm}
```

Comment:
Please move `validationSchema` and `initialValues` to a constants file in the scope of this directory.

It's better to place `validation schema` and` initialValues` for a particular form to a constant file which is in the scope of the directory that you're working in currently.
For example, consider Notes. Placing `NOTES_VALIDATION_SCHEMA` and `NOTES_INITIAL_VALUES` inside of Notes/constants.js is a better approach than putting it inside a global constant and importing it from there.

Generally, let's place constants at a global level if two or more features are going to share that constant. One basic example would be `BASE_URL`. Now, `BASE_URL` would be used at multiple places, so it makes sense to put it at a global level.